modules/ROOT/pages/upgrade-api-policies-proxies.adoc

```diff
@@ -0,0 +1,126 @@
+. Upgrade each of your API-level policies:
+. In *Policy version*, select the latest version of the included policy.
+. In *Method & resource conditions*, select either *Apply configuration to all API method & resources* or *Apply configuration to specific API method & resources*.
+. Click *Apply*.
+
+[[upgrading-custom-api-level-policies]]
+=== Upgrading Custom API-level Policies
+
+To upgrade policies at the API level that include customizations:
+
+. In API Manager, go to *API Administration > <API Name> > Policies > Add a policy* to apply policies at the individual API level. 
+. Upgrade the required extensions and connectors to Java 17.
+* If your custom policy has Anypoint Connectors, refer to: <<Upgrading Anypoint Connectors and Modules (section from J17 landing page)>>.
+* If your custom policy has custom connectors, refer to <<Upgrading Custom Connectors (section from J17 landing page)>> or <<Upgrading Rest Connect Connectors (section from J17 landing page)>>.
+. <<Annotate the custom policy (link to new page in Gateway)>>.
+. Package the custom policy.
+. Republish the custom policy to Exchange.
+. Apply the new version of the custom policy that is compatible with Java 17 to the API instance:
+ .. Go to *Policies*.
+ .. In *Applied policy status*, verify that the latest definition is applied.
+ .. If the latest definition is applied, click the three dots menu and select *Check for implementation updates*.
+ .. If the latest definition is not applied, update to the latest definition.
+. Click *Apply*.
+
+[[upgrading-api-proxies]]
+== Upgrading API Proxies
+
+The steps to upgrade API proxies to Java 17 are a little different, depending on which deployment model you use. 
+
+* If you use Basic endpoint, deploy the adapted application to the server from Mule runtime and connect it to API Manager using autodiscovery. For more information, refer to <<Update Mule runtime docs in J17 landing page>>
+* If you use a Basic endpoint API instance to update your instance, update the Mule application connecting to your API instance.
+
+[IMPORTANT]
+To ensure your API proxies or Mule apps are protected when upgrading, upgrade your API policies *before* upgrading your API proxies or Mule apps.
+
+[[cloudhub-and-cloudhub2-deployments]]
+=== CloudHub and CloudHub 2.0 deployments
+
+Before following these steps, ensure your policies are updated. For more information, see <<Upgrading Policies>>. 
+
+. In API Manager, go to *API Administration > Settings*.
+. Edit the API instance's configuration settings:  
+ .. In *Runtime Channel*, select the runtime channel to use.
+ .. In *Version*, select *4.6.0*.
+ .. In *Java version*, select *Java 17*.
+ .. Click *Save & Apply*.
+
+[[hybrid-deployments]]
+=== Hybrid deployments
+
+Before following these steps, ensure your policies are updated. For more information, refer to Upgrading Policies. 
+
+. Deploy Mule runtime engine 4.6 in a new server running on Java 17 and start the server.
```

Comment:
Could we link to the docs for how to deploy this, should be here https://docs.mulesoft.com/runtime-manager/servers-create